README.md

```diff
@@ -150,12 +150,70 @@ To additionally verify the contract on etherscan set the `ETHERSCAN_API_KEY` env
 
 Running the script will save a file with details about the deployment in `contracts/broadcast/$CHAIN_ID`.
 
-#### Folder Structure Rationale
+#### Deployment via Rust
+
+**Build and Run**
+
+```bash
+cargo run --bin deploy -- [FLAGS/OPTIONS]
+```
+
+Or, for help
+
+```bash
+cargo run --bin deploy -- --help
+```
+
+**Configuration**
+
+You can configure the deployer using CLI flags or environment variables. Most options can be set via environment
+variables (see the code for the full list `sequencer/src/bin/deploy.rs`). Common environment variables:
+
+- `ESPRESSO_SEQUENCER_L1_PROVIDER` — L1 JSON-RPC endpoint
+- `ESPRESSO_SEQUENCER_ETH_MNEMONIC` — Mnemonic for the deployer wallet
+- `ESPRESSO_SEQUENCER_ETH_MULTISIG_ADDRESS` — Multisig admin address
+- `ESPRESSO_DEPLOYER_ACCOUNT_INDEX` — Account index in the wallet
+- `ESPRESSO_SEQUENCER_URL` — Sequencer node URL for HotShot config 
+
+You can use a `.env` file and load it with:
+
+```bash
+set -a
+source .env
+set +a
+```
+
+#### Deployment via Docker
+
+You can run the deployer in a container:
+
+```bash
+docker run --env-file .env --rm ghcr.io/espressosystems/espresso-sequencer/deploy:main /bin/deploy --deploy-light-client-v1 ...
```

Comment:
Does  this actually work?

```
docker run --env-file .env --rm ghcr.io/espressosystems/espresso-sequencer/deploy:main /bin/deploy --deploy-light-client-v1
error: invalid value 'http://demo-l1-network:${ESPRESSO_SEQUENCER_L1_PORT}' for '--rpc-url <RPC_URL>': invalid port number

For more information, try '--help'.
```

Reply:
thanks updated it to the docker compose command
https://github.com/EspressoSystems/espresso-network/commit/c23f8ff53c7f715402dcb62eb31868680c009bd0